WSLBlobNFS/wsl2_linux_script.sh

```diff
@@ -301,8 +296,11 @@ function mount_share ()
                 mkdir -p $mountPath
                 vecho "Created $mountPath"
             else
-                eecho "Mount point $mountPath already exists. Use a different mount point."
-                return 1
+                # Don't allow mounts on non empty directories
+                if [[ "$(ls -1 $mountPath)" ]]; then
```

Comment:
this may also work, but let's do the following

[[ -n "$(ls -1 $mountPath)" ]]